library/src/main/java/io/andref/rx/widgets/ListViewCardAdapter.java

```diff
@@ -0,0 +1,264 @@
+package io.andref.rx.widgets;
+
+import android.content.Context;
+import android.content.res.Resources;
+import android.graphics.PorterDuff;
+import android.graphics.drawable.Drawable;
+import android.support.v4.content.res.ResourcesCompat;
+import android.support.v4.util.Pair;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageButton;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.jakewharton.rxbinding.view.RxView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import rx.Observable;
+import rx.functions.Action1;
+import rx.subjects.PublishSubject;
+
+import static android.view.View.GONE;
+import static io.andref.rx.widgets.ListViewCard.TAG;
+
+public class ListViewCardAdapter extends RecyclerView.Adapter
+{
+    private List<ListViewCard.Item> mItems = new ArrayList<>();
+
+    private PublishSubject<Pair<ListViewCard.Item, Integer>> mItemClicks = PublishSubject.create();
+    private PublishSubject<Pair<ListViewCard.Item, Integer>> mIconClicks = PublishSubject.create();
+
+    private float mCellHeight;
+    private float mAvatarAlpha;
+    private float mIconAlpha;
+    private int mAvatarTint;
+
+    private boolean mShowLastDivider = true;
+
+    public ListViewCardAdapter(float cellHeight, float avatarAlpha, float iconAlpha, int avatarTint)
+    {
+        mCellHeight = cellHeight;
+        mAvatarAlpha = avatarAlpha;
+        mIconAlpha = iconAlpha;
+        mAvatarTint = avatarTint;
+    }
+
+    @Override
+    public RecyclerView.ViewHolder onCreateViewHolder(ViewGroup parent, int viewType)
+    {
+        switch (viewType)
+        {
+            default:
```

Comment:
We are switching so that new view types can easily be added / removed. I'll drop them for now.